issues/mediacapture-surface-control-missingtask.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: Missing tasks in parallel steps in Captured Surface Control
+Tracked: N/A
+Repo: 'https://github.com/w3c/mediacapture-surface-control'
+---
+
+While crawling [Captured Surface Control](https://w3c.github.io/mediacapture-surface-control/), the following algorithms fire an event, or resolve or reject a Promise, within a step that runs [in parallel](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel) without first queuing a task:
+* [ ] The [CaptureController/setZoomLevel()](https://w3c.github.io/mediacapture-surface-control/#dom-capturecontroller-setzoomlevel) algorithm resolves/rejects a promise directly in a step that runs in parallel
+* [ ] The [CaptureController/forwardWheel()](https://w3c.github.io/mediacapture-surface-control/#dom-capturecontroller-forwardwheel) algorithm resolves/rejects a promise directly in a step that runs in parallel
+* [ ] The [forward wheel event algorithm](https://w3c.github.io/mediacapture-surface-control/#dfn-forward-wheel-event-algorithm) algorithm fires an event directly in a step that runs in parallel
```

Comment:
```suggestion
```
already tracked in https://github.com/w3c/mediacapture-surface-control/issues/51#issuecomment-2546001987